admin_guide/manage_nodes.adoc

```diff
@@ -430,14 +430,32 @@ openshift_node_kubelet_args={'max-pods': ['40'], 'resolv-conf': ['/etc/resolv.co
 [[admin-guide-max-pods-per-node]]
 === Setting Maximum Pods Per Node
 
+////
+The following section is included in the Scaling and Performance Guide.
+////
+// tag::admin_guide_manage_nodes[]
 In the *_/etc/origin/node/node-config.yaml_* file, two parameters control the
 maximum number of pods that can be scheduled to a node: `pods-per-core` and
 `max-pods`. When both options are in use, the lower of the two limits the number
-of pods on a node.
+of pods on a node. Exceeding these values can result in:
+
+* Increased CPU utilization on both {product-title} and Docker.
+* Slow pod scheduling.
+* Potential out-of-memory scenarios (depends on the amount of memory in the node).
+* Exhausting the pool of IP addresses.
+* Resource overcommitting, leading to poor user application performance.
+
+[NOTE]
+====
+In Kubernetes, a pod that is holding a single container actually uses two
+containers. The second container is used to set up networking prior to the
+actual container starting. Therefore, a system running 10 pods will actually
```

Comment:
I presume so? That's what I understand, and what I understand it to be saying. This is from the whitepaper, so unless it's changed, I'll presume this is okay.